app.js

```diff
@@ -7,6 +7,7 @@ const lookAndFeel = require('@hmcts/look-and-feel');
 const healthcheck = require('@hmcts/nodejs-healthcheck');
 const steps = require('steps');
 const paths = require('paths');
+const overviewContent = require('steps/landing-pages/overview/content.en.json');
```

Comment:
The landing pages are not steps and should be treated differently. Create a landing-page directory at the same level as steps.